plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -39,9 +47,17 @@ test.describe(
 					name: 'Your cart is currently empty!',
 				} )
 			).toBeVisible();
-			await expect(
-				page.getByRole( 'link', { name: 'Browse store' } )
-			).toBeVisible();
+			try {
+				await expect(
+					page.getByRole( 'link', { name: 'Browse store' } )
+				).toBeVisible();
+			} catch ( e ) {
+				// Cover the case when we run tests on Pressable
+				// eslint-disable-next-line jest/no-try-expect
+				await expect(
+					page.getByRole( 'heading', { name: 'New in store' } )
```

Comment:
On Pressable, there is a "New in store" element with items listed bellow.

Reply:
Is this because the theme is different?

Reply:
Yes. Do you think we should keep this change or take different approach?

Reply:
I made a mistake here. The test was failing on all themes; the link was missing. I investigated a bit and changed the template on the Pressable website. It should be fine now. I reverted this change in [87c8c73](https://github.com/woocommerce/woocommerce/pull/51014/commits/87c8c736172a02b09e24743a7d804de265d9f877) and the test is passing locally.